.github/workflows/pathogen-repo-build.yaml.in

```diff
@@ -218,6 +218,16 @@ jobs:
           | "$NEXTSTRAIN_GITHUB_DIR"/bin/json-to-envvars
           | tee -a "$GITHUB_ENV"
 
+      - name: Create runtime AWS IAM inline session policy
+        env:
+          FULL_REPO_NAME: ${{ inputs.repo }}
+        run: |
+          export REPO_NAME="${FULL_REPO_NAME#*/}"
+          export PREFIX="${REPO_NAME}_"
```

Comment:
I'm a little wary of this user input being used as-is to construct the session policy. In particular, I'm thinking about what happens if I say call this workflow with `repo: tsibley/zika`. A bunch of the generated policy happily treats that as equivalent to `nextstrain/zika`.

But, I _think_ ultimately the AWS role trust policy (WIP) would protect us there because it'll contain a condition like:

```json
"token.actions.githubusercontent.com:sub": "repo:nextstrain/*:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@refs/heads/*",
```

and the `repo:nextstrain/*:` part will not match the actual `sub` of `repo:tsibley/zika:…`.

Reply:
I'll make sure the trust policy on the AWS side enforces this.